tests/testThreads2.py

```diff
@@ -23,6 +23,7 @@
 
 import unittest
 
+from past.builtins import long
```

Comment:
It's not entirely obvious to me that `getNumThreads` really needs to return a `long`.